src/BetterAppListBuilder.php

```diff
@@ -22,107 +23,30 @@
  * USA.
  */
 
-namespace Drupal\apigee_edge_ui;
-
-use Drupal\apigee_edge\Entity\AppInterface;
-use Drupal\apigee_edge\Entity\ListBuilder\DeveloperAppListBuilderForDeveloper;
 use Drupal\Core\Entity\EntityInterface;
-use Drupal\Core\StringTranslation\TranslatableMarkup;
-use Drupal\Core\Template\Attribute;
+use Drupal\apigee_edge\Entity\ListBuilder\AppListBuilder;
 
 /**
  * Advanced list builder for developer apps.
  */
-class BetterAppListBuilder extends DeveloperAppListBuilderForDeveloper {
+class BetterAppListBuilder extends AppListBuilder {
```

Comment:
Since the functionality in this class is provided by a trait - and this PR already contains breaking changes - mark this class as final. If someone would like to customize this then s/he just needs to use the trait in a new class.

Reply:
+1 Is there anything that AppListBuilder adds to this implementation if `render()` and `getDefaultOperations()` both overridden and parent is not called? Could this be just a simple extension of the `EntityListBuilder` (or EdgeEntityListBuilder) base class?